packages/nx/src/hasher/hasher.ts

```diff
@@ -213,6 +214,9 @@ export class Hasher {
 
     performance.mark('hasher:implicit deps hash:start');
 
+    const fileHasher = defaultFileHasher;
+    await fileHasher.ensureInitialized();
```

Comment:
this should not be called here. We already invoke ensureInitialize before the hasher kicks in.